opl/generators/inventory_egress.py

```diff
@@ -29,7 +29,7 @@ def __init__(
         self.per_account_data = per_account_data
 
         # Load package profile generator
-        self.pg = opl.generators.packages.PackagesGenerator()
+        self.pg = opl.generators.packages.PackagesGenerator("package_data_rhel88.json")
```

Comment:
Instead of hardcoring this, could you add `packages_data_file="packages_data.json"` option to `__init__` instead?

Then you can configure it from iperf side